napari/_vispy/vispy_points_layer.py

```diff
@@ -9,7 +9,7 @@
 
 class VispyPointsLayer(VispyBaseLayer):
     _highlight_color = (0, 0.6, 1)
-    _highlight_width = 5
+    _highlight_width = 2
```

Comment:
Probably a separate PR, but I think we should consider moving `_highlight_color` and `_highlight_width` somewhere else so it's easier for the user to change it "on the fly" (e.g., onto the model).

Reply:
Agreed - I've added `_highlight_width` to #992 ideas for setting. Highlight color should get automatically set by our theme. I've made a new issue for that #1035

Reply:
I think theme is the right place for this, rather than the model